mantidimaging/gui/windows/live_viewer/model.py

```diff
@@ -63,28 +65,42 @@ def find_last_modified_image(self):
         Find the last modified image in the directory and
         emit the image_changed signal.
         """
-        if self.images:
-            last_modified_image = max(self.images, key=lambda x: x.stat().st_mtime)
+
+        if self.images and self._validate_images(self.images):
```

Comment:
This happens just after a call to `self._validate_images()` in `_handle_directory_change()`, I don't think we gain much from checking the files again.

This is a good case to use "Easier to ask for forgiveness than permission". Checking a file exists is no guarantee that it will still exist when we try to call `stat` or read it. Instead we should make sure we gracefully handle the failures when they happen.